pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -262,7 +262,7 @@ func TestDesiredRouterDeployment(t *testing.T) {
 			},
 			HttpLogFormat: "%ci:%cp [%t] %ft %b/%s %B %bq %HM %HU %HV",
 			HTTPCaptureCookies: []operatorv1.IngressControllerCaptureHTTPCookie{
-				{MatchType: "Prefix", NamePrefix: "foo"},
+				0: {IngressControllerCaptureHTTPCookieUnion: operatorv1.IngressControllerCaptureHTTPCookieUnion{MatchType: "Prefix", NamePrefix: "foo"}},
 			},
```

Comment:
You can omit the index.  Maybe something like this:  
```suggestion
			HTTPCaptureCookies: []operatorv1.IngressControllerCaptureHTTPCookie{{
				IngressControllerCaptureHTTPCookieUnion: operatorv1.IngressControllerCaptureHTTPCookieUnion{
					MatchType:  "Prefix",
					NamePrefix: "foo",
				},
			}},
```

Reply:
Ok, reformatted it too.